students/10th/gonholee/westagram/westagram/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from django.urls    import path,include
+
+urlpatterns = [
+    path('users',include('users.urls')),
+    path('posts',include('posts.urls')),
```

Comment:
앱은 하나기 떄문에 단수로 수정 부탁 드립니다.
```suggestion
    path('user',include('users.urls')),
    path('post',include('posts.urls')),
```